CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,49 @@
+# Code of Conduct
+
+As our headless community grows, we’re fortunate to have the ability to ask for advice, share our work, and enrich all developers in their headless pursuits. When joining our humble community, all of us become responsible for making it a safe and welcoming place we can all share with pride. The DevRel team at WPEngine is committed to making our community a place for diverse voices where all are free to express their joy of development. These guidelines help aid in achieving that goal.
+
+Therefore, remember, when interacting with other headless devs or contributing to Atlas, please: 
+
+## Stay on topic
+
+On topic has a broad meaning in Headless development, since our development work could encompass the entire stack. As long as code is involved, any development-related query is fair game. But tone is complex, hard to read. Use clear and succinct language when possible to prevent misunderstandings.
+
+And if you REALLY have a dank non-development-related meme you want to share, the #random channel in our Slack is a good place for those.
+
+## Respect
+
+Manners maketh people. Civility and professionalism are key to fostering safety. If you think a statement of yours could be offensive, abusive, or hateful, DON’T say it. Everyone deserves dignity and respect.
+
+Disagreements will happen. We do encourage you to voice them. However, disagree with the idea, not the person. No name-calling. No personal attacks. Provide counters that build instead of tear down.
+
+
+## Open-mindedness
+
+We’re all in different parts of our development journey. Some of us have years of frontline experience, others are just getting started, some have probably never written a line of code before. In this community, we welcome all ideas, no matter the source.
+
+
+## Empathetic communication
+
+Everyone has different backgrounds and perspectives that differ from yours. Before disagreeing reflexively, try to understand where the other person comes from. Ask more questions. Be comfortable responding to questions. When we empathize, we create an atmosphere of inclusiveness.
+
+
+## Ban Hammer
+
+Everyone has the right to express their ideas and challenge others. However, we have a ban hammer and we’re not afraid to use it. If you’re preventing fruitful discussion from happening by being a jerk, we’ll kick you out with no hesitation. The following items will result in immediate expulsion from the community:
+
+- Threats of violence
+- Hate speech and discrimination
+- Bullying and harassment
+- Constant disruptions
+- Impersonating another user
+- Posting others personal information
+- Sexually obscene content
```

Comment:
I read this on friday and it's kept coming to mind over the weekend. This language bothers me. At a base level it seems suggest certain expressions of sexuality are obscene. Also, I'm guessing we don't want any sexual content. This language doesn't seem to communicate that, it seems to suggest certain (non-obscene) content would be accepted.  

My suggestion would be to mirror "Violent Content". Just make it, "Sexual content". 

Thoughts?

Reply:
This makes sense to me.